docs/features/blocks/contentblocks/embedmedia.rst

```diff
@@ -0,0 +1,32 @@
+Embed Media Block
+=================
+
+The embed media block allows users to embed media like YouTube videos or SoundCloud with just a link
+to the external media source.
+
+Field Reference
+---------------
+
+Fields and purposes:
+
+* **URL** - The URL for the external media, whether it's a tweet, Facebook post, YouTube video, etc.
+
+.. note::
+    Add an external media link like this YouTube video about making fancy cupcakes: `<https://youtu.be/eXkyUi5HKpM>`_ .
+
+* **Advanced Settings** - Add custom CSS classes or a CSS ID to style the embed with your custom CSS 
+
+.. figure:: img/embedmedia1.png
+    :alt: An embed media block with fields.
+
+    An embed media block with its fields.
+
+
+.. figure:: img/embedmedia2.png
+    :alt: How it displays on the page.
```

Comment:
The result should show only the embed media. The pink bg and side column are a bit misleading.

Reply:
Yeah, I'll crop it to only show the videp. It's just the YouTube video was huge without some manipulation.

Reply:
done